R/edit.R

```diff
@@ -103,7 +104,20 @@ edit_rstudio_snippets <- function(type = c("r", "markdown", "c_cpp", "css",
   type <- tolower(type)
   type <- match.arg(type)
 
-  path <- path_home_r(".R", "snippets", path_ext_set(type, "snippets"))
+  # define path for snippets depending on
+  # 1) RStudio Version
+  # 2) OS.type
+  # https://blog.rstudio.com/2020/02/18/rstudio-1-3-preview-configuration/
+  if (rstudioapi::versionInfo()$version < "1.3") {
+    path <- path_home_r(".R", "snippets", path_ext_set(type, "snippets"))
+  } else {
+    if (.Platform$OS.type == "windows") {
+      path <- path_home_r("AppData/Roaming/Rstudio", "snippets", path_ext_set(type, "snippets"))
+    } else {
+      path <- path_home_r(".config/rstudio", "snippets", path_ext_set(type, "snippets"))
```

Comment:
I feel a little queasy hard-wiring this here. Does rstudioapi offer a way to get (more of) these paths? I'd rather consult an outside authority.

Reply:
Yes, this sounds reasonble. I couldn't find an `rstudioapi` function here. But maybe you can consult someone of the develepers!

Reply:
https://github.com/rstudio/rstudioapi/issues/176